examples/daal4py/run_examples.py

```diff
@@ -158,6 +166,13 @@ def get_exe_cmd(ex, nodist, nostream):
             return None
         if not check_library(req_library[os.path.basename(ex)]):
             return None
+    if os.path.dirname(ex).endswith("sklearnex"):
```

Comment:
i guess there is a way to rewrite this in a bit cleaner logic - for scikit you need only check for device additionally, but version check is universal for example and can be run elsewhere.

Reply:
Generally, for `sklearnex` examples I think we don't need additional restrictions.
I believe we will have some other examples, not only SPMD, and some of them on CPU as well. Here I am just checking env for exact spmd examples in `examples.sklearnex` dir.
I think here it's just required to specify condition like this:
```suggestion
    if os.path.dirname(ex).endswith("sklearnex") and not nodist and ex.endswith('spmd.py'):
```